src/main/resources/application.yml

```diff
@@ -20,10 +20,79 @@ management:
 api:
   context-path: "/api"
 
+security:
+  basic:
+    enabled: \${SECURITY_ENABLED:true}
```

Comment:
The escaping slashes on Spring property references are due to our use of Gradle's "expand" function which complains if it finds an unknown property reference.

An old spring boot commit explains that Spring made this change expressly for Gradle support:
https://github.com/spring-projects/spring-boot/commit/c0c67f2593dbfd17aa304b43f4da3a3678fa58eb